.github/workflows/build-status-check.yml

```
@@ -15,29 +15,38 @@ on:
 jobs:
   build_main_project:
     name: Build ${{ vars.PROJECT_NAME }}
-    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.0.1
+    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.1.0
     with:
       project-name: "${{ vars.PROJECT_NAME }}"
       build-config: Debug
       net-sdk-version: "${{ vars.NET_SDK_VERSION }}"
+      checkout-ref: ${{ github.event.pull_request.head.ref }}
+      checkout-repository: ${{ github.event.pull_request.head.repo.full_name }}
+
 
 
   build_perf_project:
     name: Build ${{ vars.PROJECT_NAME }}PerfTests
-    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.0.1
+    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.1.0
     with:
       project-name: "${{ vars.PROJECT_NAME }}PerfTests"
       build-config: Debug
       net-sdk-version: "${{ vars.NET_SDK_VERSION }}"
+      checkout-ref: ${{ github.event.pull_request.head.ref }}
+      checkout-repository: ${{ github.event.pull_request.head.repo.full_name }}
+
```

Please remove the extra line so that end result is only 2 empty lines between jobs

```suggestion
      checkout-ref: ${{ github.event.pull_request.head.ref }}
      checkout-repository: ${{ github.event.pull_request.head.repo.full_name }}
```